sapjcoserver/src/main/resources/application.yaml

```diff
@@ -0,0 +1,32 @@
+camel:
+    springboot:
+        main-run-controller: 'true'
+
+sap:
+  system:
+    name: TESTSAP
+jco:
+    client:
+        ashost: 192.180.32.18

```

Comment:
It could be useful to get this propery from the system env variables just like `{JCO_ASHOST}`

Reply:
It is already modified as you suggested